akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -135,9 +193,21 @@ final case class `If-Range`(entityTagOrDateTime: Either[EntityTag, DateTime]) ex
 }
 
 // FIXME: resurrect SSL-Session-Info header once akka.io.SslTlsSupport supports it
-final case class RawHeader(name: String, value: String) extends jm.headers.RawHeader {
+final class RawHeader(val name: String, val value: String) extends jm.headers.RawHeader {
```

Comment:
True, thanks. I'll see if possible to keep it a case class and just add the apply/unapply manually.